apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -65,6 +203,9 @@ export function VariantsPage() {
         onScrollPositionChange={setScrollPosition}
       >
         {variants?.map((variant, variantIndex) => {
+          const isActiveError = errorState?.variantIndex === variantIndex;
+          const errorMessage = isActiveError ? errorState.errorMessage : variantsErrorMap?.[`variant-${variantIndex}`];
```

Comment:
since, varaints can have multiple errors and we want to cycle through them while navigating, this will enable to sync the error message in the error navigation, variant node and the popover, so that same error message is being shown in these 3 places

Reply:
The ``variant-${variantIndex}`` key identifier is being used in a few places, could we extract to a helper function `getVariantErrorKey(index: number)` for maintainability?